### DISCUSSION TITLE
Feature Request - add icon to dll?

### DISCUSSION THREAD

**Discussion by PGomersall** [valinet/ExplorerPatcher#311] [General]
Can we get an icon added to the dll so when when\if a shortcut is created via rundll32 we can then point it at the included icon?
I was thinking something like:
![ExplorerPatcher](https://user-images.githubusercontent.com/43388507/141183070-93872c93-c2ea-4dbd-a21f-8d97e548c9eb.png)

**Comment by valinet** [comment]
You can use the icon that's currently grabbed by the Settings window from `shell32.dll`, number 40:

![image](https://user-images.githubusercontent.com/92230793/137664826-54dae64e-2476-4cf7-9d65-e0f6993cda5f.png)

Native looking icons are copyrighted, that's the main issue. The icon you posted is a derivative of copyrighted work. By loading it from `shell32.dll`, I do not distribute copyrighted stuff, for which I do not have a license to, but rather use what's already on your computer. Second issue is that graphics that looks good, scales good etc takes quite some time to create and maintain; I am no graphic designer, I am not interested in investing anything but the bare minimum amount of time on it; if I can find a quick hack to load a decent icon that fits, I use that and call it a day.

Thanks

**Reply by PGomersall** [reply]
Understood Valinet
